src/server.js

```diff
@@ -126,43 +160,58 @@ app.get("/logout", function (req, res) {
 });
 
 app.use(function (req, res, next) {
-    var { requestUrl, namespaceComputed } = getUserInput(req);
+    var {
+        requestUrl,
+        namespaceComputed
+    } = getUserInput(req);
     if (requestUrl) {
         userInputNamespace = namespaceComputed;
         var fileName = "/summary" + requestUrl;
         if (fs.existsSync(__dirname + fileName)) {
             res.sendFile(__dirname + fileName);
-        }
-        else {
+        } else {
             res.sendFile(__dirname + requestUrl);
         }
     }
 });
 
-function processCommands(req, res, command) {
-    var { requestUrl, namespaceComputed } = getUserInput(req);
+function processCommands(req, res, command, contentType = "application/json") {
+    var {
+        requestUrl,
+        namespaceComputed
+    } = getUserInput(req);
     userInputNamespace = namespaceComputed;
     if (userInputNamespace && command) {
-        var selector = getLabelSelector(req);
-        var promise = selector ? command(userInputNamespace, "false", selector) : command(userInputNamespace, "false");
+        var selector = getQueryParameterValue(req, "labelselector");
+        var podName = getQueryParameterValue(req, "podName");
+        var promise = selector ?
+            command(userInputNamespace, "false", selector) :
+            podName ? command(userInputNamespace, "false", podName) : command(userInputNamespace, "false");
```

Comment:
Should we handle podName here, or handle get pod logs separately?

Reply:
it worked, i did not check in deep.